cupy/misc/who.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import cupy as cp
+
+
+def who(vardict=None):
+
+    """
+    Print the CuPy arrays in the given dictionary.
```

Comment:
docstring does not align with CuPy format

Reply:
Could you please point me to somewhere that explains the difference? I'm not sure what the Google style is

Reply:
You can check other routines in the code.